torch/csrc/stable/accelerator.h

```diff
@@ -0,0 +1,69 @@
+#pragma once
+
+#include <torch/csrc/inductor/aoti_runtime/utils.h>
+
+#include <torch/csrc/inductor/aoti_torch/c/shim.h>
+
+namespace torch::stable::accelerator {
+
+namespace {
+inline void delete_device_guard(void* ptr) {
+  AOTI_TORCH_ERROR_CODE_CHECK(
+      aoti_torch_delete_device_guard(reinterpret_cast<DeviceGuardHandle>(ptr)));
+}
+
+} // namespace
+
+using DeviceIndex = int8_t; // this is from c10/core/Device.h
```

Comment:
We should NOT rely on this one at the shim level.
For this part of the world, let's make it an int32_t for consistency with the existing shim.
We will have to change the tensor.h version.